models/columnboolean.go

```diff
@@ -0,0 +1,52 @@
+package models
+
+import (
+    "encoding/json"
+    "errors"
+)
+
+// ColumnBoolean Model
+type ColumnBoolean struct {
+    // Column Key.
+    Key string `json:"key"`
+    // Column type.
+    Type string `json:"xtype"`
+    // Column status. Possible values: `available`, `processing`, `deleting`,
+    // `stuck`, or `failed`
+    Status string `json:"status"`
+    // Error message. Displays error generated on failure of creating or deleting
+    // an column.
+    Error string `json:"xerror"`
```

Comment:
similar here, I dont think json keys have x-es. we should remove this "x"-ing logic from json keys on response models